internal/pkg/text/template/parse/node.go

```diff
@@ -193,6 +214,23 @@ func (p *PipeNode) writeTo(sb *strings.Builder, isConditional bool) {
 		c.writeTo(sb, isConditional)
 	}
 }
+func (p *PipeNode) writeForTo(sb *strings.Builder) {
```

Comment:
Please document this method and the fact that it is an aberration from the standard "writeTo(*strings.Builder)" API.  This is a good;  it is a great way to override the behavior since the language is limited in this respect.  I like that you put it in a renamed method since it keeps the rest of things intact.  However, we just want to be clear in case someone else picks this up in the future and wants to see what we did.